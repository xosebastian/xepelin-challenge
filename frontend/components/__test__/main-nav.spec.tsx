import { render, screen } from "@testing-library/react";
import { MainNav } from "../main-nav";


describe("MainNav", () => {
  it("should render the Xeplin Bank link", () => {
    const { container } = render(<MainNav />);
    expect(container).toMatchSnapshot();
    expect(screen.getByText("Xeplin Bank")).toBeDefined();
  });
});