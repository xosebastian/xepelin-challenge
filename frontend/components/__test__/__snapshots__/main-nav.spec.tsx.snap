// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainNav should render the Xeplin Bank link 1`] = `
<div>
  <nav
    class="flex items-center space-x-4 lg:space-x-6"
  >
    <a
      class="text-sm font-medium transition-colors hover:text-primary"
      href="/examples/dashboard"
    >
      Xeplin Bank
    </a>
  </nav>
</div>
`;
